### ISSUE TITLE
Cropper Color

### ISSUE THREAD

**Issue by laura-chesches** [CrackerakiUA/ui-cropper#67]
Hello, 
Can you please help me change the cropper's color and the background? I can't do it with plain CSS because it's in a canvas.
![image](https://user-images.githubusercontent.com/30465251/35672353-efb6688c-0746-11e8-9f5e-dedd2abf5fb9.png)

Thanks

**Comment by CrackerakiUA** [comment]
Yes, with css is not possible, you have to modify plugin. At the moment i don't have much free time to add such option. If you planning to add such option by your self, i can guide you.

**Comment by rickerd** [comment]
It’s build with JS and very static.
If you want to edit such a thing you’ll need to build a feature.

**Comment by CrackerakiUA** [comment]
It can be cool feature:
1) Custom background
2) Custom controllers
3) Custom crop area
